js/tidelinedata.js

```diff
@@ -531,24 +531,26 @@ function TidelineData(data, opts) {
   var sortedParameters =_.orderBy(parameters,['normaltime'], ['desc']);
```

Comment:
```suggestion
  var parameters = _.filter(data, {type: 'deviceEvent', subType: 'deviceParameter'});
  var sortedParameters =_.orderBy(parameters, ['normaltime'], ['desc']);
```
Can make a comment on the line above.